samcli/hook_packages/terraform/hooks/prepare/resource_linking.py

```
@@ -99,13 +99,62 @@ def _get_configuration_address(address: str) -> str:
     Cleans all addresses of indices and returns a clean address
 
     Parameters
-    ==========
+    ----------
     address : str
         The address to clean
 
     Returns
-    =======
+    -------
     str
         The address clean of indices
     """
     return re.sub(r"\[[^\[\]]*\]", "", address)
+
+
+def _resolve_module_output(module: TFModule, output_name: str) -> List[Union[ConstantValue, ResolvedReference]]:
+    """
+    Resolves any references in the output section of the module
+
+    Parameters
+    ----------
+    module : Module
+        The module with outputs to search
+    output_name : str
+        The value to resolve
+
+    Returns
+    -------
+    List[Union[ConstantValue, ResolvedReference]]
+        A list of resolved values
+    """
+    results: List[Union[ConstantValue, ResolvedReference]] = []
+
+    output = module.outputs[output_name]
```

use get instead of `[]`, and please raise an exception if it is not there